src/lib/var_export_html.c

```diff
@@ -231,11 +237,14 @@ void xdebug_var_export_html(zval **struc, xdebug_str *str, int level, int debug_
 				xdebug_str_add(str, xdebug_sprintf("<i>&amp;</i><b>object</b>(<i>%s</i>)", STR_NAME_VAL(Z_OBJCE_P(*struc)->name)), 1);
 				xdebug_str_add(str, xdebug_sprintf("[<i>%d</i>]\n", Z_OBJ_HANDLE_P(*struc)), 1);
 			}
+#if PHP_VERSION_ID >= 70400
+			zend_release_properties(myht);
+#else
 			if (is_temp) {
 				zend_hash_destroy(myht);
 				efree(myht);
 			}
```

Comment:
Is there a reason why this doesn't use `xdebug_var_maybe_destroy_ht`, which is used in `var_export_xml.c`, but not in the other `var_export_*.c` files? If there is no reason why it can't be used, I'll switch it out before merging.